### ISSUE TITLE
Gradient not working

### ISSUE THREAD

**Issue by VardanHeroic** [nikp123/xava#54]
Color part of my config

```
background = 'black'
foreground = '#5106db'

# This adjusts the opacity of the bars (must have transparency for it to work)
; foreground_opacity = 1.0

# And this does the same but for the background (under the same condition as above)
; background_opacity = 1.0

# Gradient mode, only hex defined colors are supported, background must also be
# defined in hex or remain commented out.
# To enable gradients, just change gradient_count to the number of colors you
# have specified (0 = disabled)
gradient_count = 2
gradient_color_1 = '#5106db'
gradient_color_2 = '#ff1593'
```

![image](https://github.com/nikp123/xava/assets/106673994/9713690d-f3e4-42cd-86a7-4f8673529847)

**Comment by nikp123** [comment]
Yeah I didn't migrate that code. Will do when I get available time.

**Comment by UserNameBlank** [comment]
fixed with #58
